src/cli.ts

```diff
@@ -2,37 +2,83 @@
 
 import path from "path";
 import fs from "fs";
+import yargs from "yargs/yargs";
 
 import { FileServer } from "./file-server";
 import { prepareAppOptions } from "./prepare-app-options";
 import { captureScreenshot } from "./capture-screenshot";
+import {
+  DEFAULT_WIDTH,
+  DEFAULT_HEIGHT,
+  DEFAULT_FORMAT,
+  DEFAULT_QUALITY,
+  DEFAULT_TIMEOUT_MILLISECONDS,
+  DEFAULT_DEBUG,
+  DEFAULT_VERBOSE_LOGGING,
+} from "./settings";
```

Comment:
I noticed we were pulling in defaults in another file. I brought these to be constants and made `yargs` handle these below so it's a bit easier to grok where these default settings come from.